### PR TITLE
Additional probes, increase slave timeout

### PR DESCRIPTION
**Pull request by deveshmanish**
* Increased Slave polling timeout from 60s to 300s to give slaves ample time to poll multiple targets.
    * Added additional probes: 
        * speedtest.pm - Ookla official speedtest client
        * speedtest-cli.pm - Sievel's speedtest-cli
        * YoutubeDL.pm - YoutubeDL probe
        * WifiParam.pm - WiFi parameter probe with iw

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.7.3
+2.7.4
```
Comment (oetiker) [inline VERSION:1]:
```suggestion
2.7.3
```
please do not change the release

File: lib/Smokeping.pm [hunk 2]
```diff
@@ -74,7 +74,7 @@ use Smokeping::RRDtools;
 # global persistent variables for speedy
 use vars qw($cfg $probes $VERSION $havegetaddrinfo $cgimode);
 
-$VERSION="2.006000";
+$VERSION="2.7.4";
```
Comment (oetiker) [inline lib/Smokeping.pm:77]:
```suggestion
$VERSION="2.006000";
```

please do not change the release

**Review by oetiker — CHANGES_REQUESTED** [review]

**Comment by oetiker** [comment]
I see that the additional probes have been created by @mad-ady. Wondering why he is not contributing them himself.

**Comment by deveshmanish** [comment]
> I see that the additional probes have been created by @mad-ady. Wondering why he is not contributing them himself.

@mad-ady might be able to shed some light on that. In view of getting the process expedited, I had proposed this PR as I have been using this patched work for long and hoped this would make things simpler.

**Comment by mad-ady** [comment]
Sorry - I didn't contribute them because I didn't think they would be useful for the general public, so I kept them separate. Over the years some people asked questions about speedtest especially. 

The probes (speedtest/youtube-dl) depend on external binaries some of which might change often and need updating, and also, the probes themselves bypass the normal smokeping measurement process by doing only one measurement and reporting 3 identical measurements. This is in order to prevent too much resource utilization/getting banned on those services for being too aggressive. There are also some limitations with regard to the unit of measurement reported. By default smokeping likes to measure time on the y axis, but these probes measure traffic/DB/MHz. If @oetiker doesn't mind with this hack, then fine!

I don't mind having this PR merged, even if it's not submitted by me. Though I'm not sure if the code is from my repos directly, or had some changes.

Cheers!

**Comment by oetiker** [comment]
@mad-ady reporting the same measurment multiple times seems rather odd to me ... how about complaining if pings is more than 1?

**Comment by mad-ady** [comment]
Unfortunately, that won't work because smokeping itself has a minimum limit
of 3 pings. Lowering that might break other internal processing, so I chose
to fake it :)

**Pull request closed without merging**